app/controllers/journals_controller.rb

```diff
@@ -2,7 +2,27 @@ class JournalsController < ApplicationController
 
   def index
     @journals = Journal.all
-    
+
+  end
+
+  def edit
+    @journal = Journal.find(params[:id])
+  end
+
+  def update
+    @journal = Journal.find(params[:id])
+    @journal.assign_attributes(journal_params)
```

Comment:
There is no need to do a separate `assign_attributes` and then `save` call. That's only necessary when you need to manually make other changes to the model in addition to the mass-assignment. Typically you'll just call 

```
if @journal.update(journal_params)
   ...
```
after the find